### DISCUSSION TITLE
Codeblock title with file icon

### DISCUSSION THREAD

**Discussion by A3Bagged** [squidfunk/mkdocs-material#7772] [Q&A] [answered]
Hello, i was wondering how i could make custom codeblocks that had the filename but also a file icon like the one on the actual site.
I think i can do it myself by CSS but maybe it's already implemented? I couldnt find it so i think i'll have to custom code it.

Example can be seen [here](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#annotation-tooltip-width)

![afbeelding_2024-12-05_210848331](https://github.com/user-attachments/assets/5ad9e0ff-443e-45f2-809a-29b750f2668e)

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @A3Bagged,
you can check the source of the viewed site:
- https://github.com/squidfunk/mkdocs-material/blob/6c796c06ecc0fb641ad302c784822139c2d3807a/docs/reference/code-blocks.md?plain=1#L523

Basically normal title with Markdown inside. I don't think there is anything more needed to be enabled 🤔

**Reply by A3Bagged** [reply]
> Hello @A3Bagged, you can check the source of the viewed site:
> 
> * https://github.com/squidfunk/mkdocs-material/blob/6c796c06ecc0fb641ad302c784822139c2d3807a/docs/reference/code-blocks.md?plain=1#L523
> 
> Basically normal title with Markdown inside. I don't think there is anything more needed to be enabled 🤔

I feel dumb 🤣 Sorry, kinda new to github and mkdocs, i thought to ask before i try and code it myself.
